### PR TITLE
Document tensor rendering options

### PR DESCRIPTION
**Pull request by bratseth**
Please review, but we should hold off merging for a week or so.
This functionality is expected to be released 2022-01-17.

### CODE REVIEW

File: en/reference/document-v1-api-reference.html [hunk 1]
```diff
@@ -474,7 +474,56 @@ <h2 id="request-parameters">Request parameters</h2>
     The HTTP status code will always be 200 if the visit is successfully initiated. Default value is false.
     </p>
   </td>
-</tr><tr>
+</tr>
+<tr>
+  <th>tensor.format</th>
```
Comment (lesters) [inline en/reference/document-v1-api-reference.html:479]: format.tensors?

File: en/stateless-model-evaluation.html [hunk 2]
```diff
@@ -70,13 +70,87 @@ <h2 id="model-inference-using-the-rest-api">Model inference using the REST API</
...
+<table class="table">
+    <thead>
+    <tr>
+        <th>Parameter</th>
+        <th>Type</th>
+        <th>Description</th>
+    </tr>
+    </thead><tbody>
+<tr>
+    <th>tensor.format</th>
```
Comment (lesters) [inline en/stateless-model-evaluation.html:94]: format.tensors?

**Review by lesters — APPROVED** [review]
👍

File: en/stateless-model-evaluation.html (hunk 2 above)
Comment (bratseth) [reply en/stateless-model-evaluation.html:94]: Yes, thanks!

**Pull request merged**